package.json

```diff
@@ -10,20 +10,20 @@
   },
   "repository": {
     "type": "git",
-    "url": "git+https://github.com/Unity-CI/versioning-backend.git"
+    "url": "git+https://github.com/Game-CI/versioning-backend.git"
```

Comment:
Our GitHub handle is `game-ci`, in lowercase (also see other occurrences that you've updated)